hummingbot/connector/exchange/zebpay/zebpay_utils.py

```diff
@@ -0,0 +1,44 @@
+from hummingbot.client.config.config_var import ConfigVar
+from hummingbot.client.config.config_methods import using_exchange
+from hummingbot.core.utils.tracking_nonce import get_tracking_nonce
+
+
+CENTRALIZED = False
+
+USE_ETHEREUM_WALLET = False
+
+EXAMPLE_PAIR = "DAI-INR"
+
+DEFAULT_FEES = [0.15, 0.25]
```

Comment:
I don't find these numbers in the documentation... If it's a guess we should add a `#todo` saying that this numbers are just a guess.

Reply:
I actually looked it up here:
https://zebpay.com/in/fees-old/

Granted, the URL says "fees-old" so may they updated since? I'll use those as placeholders until we confirm. Just need to add that to our questions for them.

Reply:
Ah, very interesting, I wasn't aware of this info. thanks! I will make a note of this.